interface/patient_file/encounter/forms.php

```diff
@@ -687,7 +686,7 @@ function myGetRegistered($state = "1", $limit = "unlimited", $offset = "0")
     $StringEcho .= "</ul>" . $StringEcho2;
 }
 ?>
-<table cellspacing="0" cellpadding="0" align="center">
+<table class="mx-auto">
```

Comment:
Add bootstrap table class and add table-responsive div unless this ends up breaking the code

Reply:
Hi @tywrenn 

If we add the bootstrap table and table-responsive class in here, it will cause the dropdown menu to have an inner scroll issue.

![image](https://user-images.githubusercontent.com/22230889/89725336-0af85e00-da41-11ea-8b55-c7f5fa83421d.png)

Reply:
Good catch -- add a comment on that line saying that this has a styling exception, something like "Can't have BS4 classes here or will mess up navbar"

Reply:
Done!